ssl/statem/extensions_srvr.c

```diff
@@ -154,7 +154,7 @@ int tls_parse_ctos_srp(SSL *s, PACKET *pkt, unsigned int context, X509 *x,
      * upon resumption. Instead, we MUST ignore the login.
      */
     if (!PACKET_strndup(&srp_I, &s->srp_ctx.login)) {
-        *al = TLS1_AD_INTERNAL_ERROR;
+        *al = SSL_AD_INTERNAL_ERROR;
```

Comment:
For consistency. All the alert codes have a protocol specific version indicating the protocol in which they were first defined, e.g. TLS1_AD_INTERNAL_ERROR, TLS1_AD_UNKNOWN_CA, SSL3_AD_BAD_CERTIFICATE etc, and a non-protocol specific version, e.g. SSL_AD_INTERNAL_ERROR, SSL_AD_UNKNOWN_CA, SSL_AD_BAD_CERTIFICATE. There is also a protocol specific translation function (e.g. see ssl3_alert_code()) which translates any given alert code into one specific to that protocol version. Mostly this just returns what you pass in, except in the case where a specific alert does not exist in that protocol version. If the alert code is not defined in that protocol version then it translates it into some other appropriate alert code that is defined.

I believe the intention is that we should always use the non-protocol specific version. At least that's what we use pretty much exclusively at the moment with the exception of the few items that I am changing here. I'm just trying to make it consistent.

Probably we don't really need the protocol specific defines - but because they are in public header files it is difficult to get rid of them.